### DISCUSSION TITLE
Inclusive French

### DISCUSSION THREAD

**Discussion by 1000i100** [argosopentech/argos-translate#46] [Ideas]
I wish to propose my website in French inclusive and in French academic.
I could do my home translation routines or make translations by hand, but by discovering argos-translate, I say that making a reusable automatic translator would benefit feminist communities.

So do you think it's feasible? (I hope so)

Do you have any suggestions for language naming for these French variants?
naively what comes to me is:

- fra-ac (académique) : français pratiqué par **tous ceux** qui acceptent que le masculin invisibilise le féminin comme appris à l'école.
- fra-isn (inclusif sans néologisme) : français pratiqué par **toutes celles et tous ceux** qui n'acceptent pas que le masculin invisibilise le féminin
- fra-i (inclusif) : français pratiqué par **tou·te·s celleux** qui n'acceptent pas que le masculin invisibilise le féminin
- fra-fn (féminin neutre) : français pratiqué par **toutes celles** qui n'acceptent pas que le masculin invisibilise le féminin

Is there somebody that wants to get involved in this project?

**Comment by PJ-Finlay** [comment]
Sounds like an interesting project! 

My understanding is that you want to be able to translate to and from variants of French with Argos Translate. The way to do this is to gather data for that variant. If you gather data I can translate models for you.

I suggest you use codes to not conflict with ISO 639 spec: https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes

**Comment by mayeulk** [comment]
Hi, any update on this? A related task could be: how to translate a text mentioning a non-binary person. For instance, the page of Erica Chenoweth (a famous political scientist) reads:

> A word on pronouns: I am pretty indifferent to pronouns and don’t strongly identify with any of them. If pressed, I prefer being called by my name or they/them; however, I am not offended when people use she/her or he/him.

Source: https://www.ericachenoweth.com/cv
On the www.ericachenoweth.com page, E. Chenoweth is always referred to as "they/their", never she/her or he/him.

The following sentence:
> Erica Chenoweth is a researcher. They live in the United States.

Gets translated this way by libretranslate.com :
> Erica Chenoweth est une chercheuse. Ils vivent aux États-Unis.

Clearly, "ils" (for plural meaning of they) is wrong, as Erica Chenoweth is a single person.
It could be (non-binary French flavour):
> Erica Chenoweth est un.e chercheur.euse. Iel vit aux États-Unis.

Similarly:
> Erica Chenoweth est un.e chercheur.euse. Iel vit aux États-Unis.

Gets translated this way by libretranslate.com :
> Erica Chenoweth is a researcher. He lives in the United States.

It could be (non-binary English flavour):
> Erica Chenoweth is a researcher. They live in the United States.

**Reply by PJ-Finlay** [reply]
The best way to handle something like this would be to create training data with the desired translation. You can submit data to LibreTranslate by using the edit button on a translation at LibreTranslate.com

**Reply by mayeulk** [reply]
Thank you for the hint. I have submitted the four "Chenoweth" sentences to LibreTranslate by using the edit button.

**Comment by mayeulk** [comment]
@1000i100 : note there are useful sections on gender-neutral writing in software translation, in:
 https://di.uqo.ca/id/eprint/1334/1/Pigeon_Julie_2021_travail_dirige.pdf
GUIDE PRATIQUE DE LA LOCALISATION VIDÉOLUDIQUE EN FRANÇAIS CANADIEN, PRÉCÉDÉ DE L’ÉTAT DES LIEUX
Travail dirigé professionnel présenté par: Julie Pigeon
Juillet 2021
UNIVERSITÉ DU QUÉBEC EN OUTAOUAIS. Département des études langagières

**Comment by JuPigeon** [comment]
Je remercie le professeur Kauffmann de m'avoir invitée à rejoindre cette discussion fort intéressante, car il faut en effet pouvoir nommer ces nouvelles variantes avec des codes ISO. J'ai regardé les suggestions de 1000i100, et j'ai quelques commentaires.

Tout d'abord, il serait important de conserver la variété régionale dans le code ISO (fra-ca, fra-fr, fra-be, etc.), car l'application de certains principes d'écriture inclusive pourrait varier d'une région linguistique à une autre. Le code d'écriture inclusive ne devrait donc pas effacer les variantes régionales. Par exemple, si un client me demande de traduire en fra-ac, je dois quand même connaître le public cible (comme la France ou le Canada).

Si l'on suit le principe des codes ISO pour le moyen et l'ancien français (respectivement frm et fro), nous pourrions donc modifier la première partie du code, tout en laissant la variante régionale dans la seconde partie. Nous sommes cependant limités à trois lettres pour la première partie du code.

Pour ce qui est des définitions, je ne suis pas tout à fait d'accord avec la manière négative dont elles sont présentées (« qui n'acceptent pas que... »). Je crois qu'une reformulation positive serait souhaitable. De plus, l'emploi du terme « académique » pour qualifier le français où le masculin l'emporte me donne l'impression que les autres variantes ne seraient pas acceptables dans un contexte académique, ce qui n'est pas tout à fait juste. Je dirais plutôt « masculin générique ». Je dirais également « féminin générique » au lieu de « féminin neutre », car la langue n'est pas plus neutre si le féminin l'emporte.

D'autre part, il y a encore beaucoup de discussions quant aux appellations des diverses variantes. Selon l'article _Les personnes non-binaires en français : Une perspective concernée et militante_ d'Ashley (2019), le français inclusif est un français qui ne « reconnaît pas l’existence des personnes non-binaires », alors que le français neutre le reconnaît (avec néologismes dans les pronoms).

Sur la base de ces réflexions, voici ce que je proposerais :
fmg-XX : français où le masculin l'emporte (masculin générique)
ffg-XX : français où le féminin l'emporte (féminin générique)
frn-XX : français neutre, où l'emploi de néologismes et de nouveaux codes (point médian) est accepté (tou·te·s celleux)
fri-XX : français inclusif, qui privilégie l'emploi de formulations épicènes et de doublets complets (toutes celles et tous ceux)

(Le XX serait remplacé par le code de la région, fr, ca, be, etc.)

Je suis bien entendu ouverte à vos commentaires sur mes suggestions.

**Reply by mayeulk** [reply]
Thank you for this detailed, very informative message! I like these:
```
frn-XX : français neutre
fri-XX : français inclusif
```
I see a potential issue with:
```
fmg-XX : français où le masculin l'emporte (masculin générique)
ffg-XX : français où le féminin l'emporte (féminin générique)
```
The issue is to propose a coding scheme that could work for other ("grammatically-gendered") languages.
fmg- and ffg- are based on the idea the 'f' is enough to remind us it's French. With such a rule, we can at best code 26 languages in the world. fro- and frm- keep the first 2 letters of French, which I believe can help.

So my suggestion would be:
```
frf-XX : français où le féminin l'emporte (féminin générique)
frm-XX : français où le masculin l'emporte (masculin générique)
```
(There might stil be an issue with Frisian languages, coded frr, frs, fry; I'm unsure but it might seem they have [a common gender](https://qr.ae/pKHOda), so might not need an additional code.)

Also, I have a question: Are frm-XX and fra-XX different? If not, we do not need frm-XX.

(Reference: https://fr.wikipedia.org/wiki/Liste_des_codes_ISO_639-2#f )
